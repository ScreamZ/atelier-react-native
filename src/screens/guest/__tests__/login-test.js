import React from 'react'
import LoginScreen from '../Login'

import renderer from 'react-test-renderer'

test('Is home rendered correctly', () => {
  const tree = renderer.create(<LoginScreen />).toJSON()
  expect(tree).toMatchSnapshot()
});
