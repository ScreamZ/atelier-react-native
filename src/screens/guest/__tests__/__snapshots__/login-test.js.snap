// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Is home rendered correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <Text>
    Hello login
  </Text>
</RCTSafeAreaView>
`;
